daq/session_server.py

```diff
@@ -20,34 +23,43 @@
 DEFAULT_SERVER_PORT = 50051
 DEFAULT_BIND_ADDRESS = '0.0.0.0'
 DEFAULT_SERVER_ADDRESS = '127.0.0.1'
-DEFAULT_RPC_TIMEOUT_SEC = 10
+DEFAULT_RPC_TIMEOUT_SEC = 600
```

Comment:
why is it 600? it seems to be a long time to wait for 1 request.

Reply:
That value wasn't really used -- it was only for the testing client. I just took it out completely.